.travis.yml

```diff
@@ -1,21 +1,24 @@
 language: node_js
 node_js:
-- '11.14.0' # pin to stable version
+- 11.14.0
 cache: yarn
 before_install:
 - if [[ -n "$encrypted_5b7b89e871be_key" ]]; then openssl aes-256-cbc -K $encrypted_5b7b89e871be_key
   -iv $encrypted_5b7b89e871be_iv -in .travis/.service-account-key.json.enc -out .service-account-key.json
   -d; fi
-# install specific version of yarn
+  # install specific version of yarn
 - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.15.2
 - export PATH="$HOME/.yarn/bin:$PATH"
+- echo $import_ui_data_pwd | gpg --passphrase-fd 0 .travis/serviceAccountKey_tests.json.gpg
```

Comment:
@DurkoMatko what was your thinking behind using PGP here? vs the openssl mechanism that was being used for the other .service-account-key ?

Reply:
I wasn't able to make the openssl work :/ I know it's not ideal to do same thing in two different ways but I just felt ready to take anything what makes the whole test story closer to an end.